test/utils/structs/EnumerableSetExtended.test.js

```diff
@@ -24,19 +24,19 @@ async function fixture() {
             add: `$add(uint256,${value.type})`,
             remove: `$remove(uint256,${value.type})`,
             contains: `$contains(uint256,${value.type})`,
-            clear: `$clear_EnumerableSetExtended_${name}(uint256)`,
-            length: `$length_EnumerableSetExtended_${name}(uint256)`,
-            at: `$at_EnumerableSetExtended_${name}(uint256,uint256)`,
-            values: `$values_EnumerableSetExtended_${name}(uint256)`,
-            valuesPage: `$values_EnumerableSetExtended_${name}(uint256,uint256,uint256)`,
+            clear: `$clear(uint256)`, // `$clear_EnumerableSetExtended_${name}(uint256)` when more types are added
+            length: `$length(uint256)`, // `$length_EnumerableSetExtended_${name}(uint256)` when more types are added
+            at: `$at(uint256,uint256)`, // `$at_EnumerableSetExtended_${name}(uint256,uint256)` when more types are added
+            values: `$values(uint256)`, // `$values_EnumerableSetExtended_${name}(uint256)` when more types are added
+            valuesPage: `$values(uint256,uint256,uint256)`, // `$values_EnumerableSetExtended_${name}(uint256,uint256,uint256)` when more types are added
```

Comment:
Do we still need these?

Reply:
Are you referring to the comments like _"`$clear_EnumerableSetExtended_${name}(uint256)` when more types are added"_?

I think more types may appear in EnumerableExtended, but can't say for sure. I would keep the comments.